### PR TITLE
bump libs for WebVR API v1.0

### PR DESCRIPTION
**Pull request by cvan**
see https://github.com/aframevr/aframe-core/pull/796

### CODE REVIEW

File: src/lib/three.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ require('../../node_modules/three-dev/examples/js/loaders/OBJLoader');  // THREE
 require('../../node_modules/three-dev/examples/js/loaders/MTLLoader');  // THREE.MTLLoader
 require('../../node_modules/three-dev/examples/js/loaders/ColladaLoader');  // THREE.ColladaLoader
 require('../../vendor/Raycaster');  // THREE.Raycaster
-require('../../node_modules/three-dev/examples/js/controls/VRControls');  // THREE.VRControls
-require('../../node_modules/three-dev/examples/js/effects/VREffect');  // THREE.VREffect
+require('../../node_modules/webvr-libs/VRControls');  // THREE.VRControls
```
Comment (cvan) [inline src/lib/three.js]: when this is merged upstream and three-dev is bumped, we'll point back to @ngokevin's

File: package.json [hunk 2]
```diff
@@ -35,7 +35,7 @@
     "style-attr": "^1.0.2",
     "three-dev": "^0.74.0-beta8",
     "tween.js": "^15.0.0",
-    "webvr-polyfill": "borismus/webvr-polyfill#3f47796"
```
Comment (cvan) [inline package.json]: when Brandon's PR is merged upstream, we'll point back to Boris'

File: src/lib/three.js (hunk 1 above)
Comment (ngokevin) [inline src/lib/three.js]: Crap, these probably don't work because we bumped to three.js official. This should be `s/three-dev/three/g`!

Comment (ngokevin) [inline src/lib/three.js]: Wait, why does this diff say ```three-dev```? It should be `three`.

Comment (cvan) [inline src/lib/three.js]: sorry, gotta rebase

File: src/lib/three.js [hunk 3]
```diff
@@ -23,7 +23,7 @@ require('../../node_modules/three/examples/js/loaders/OBJLoader');  // THREE.OBJ
 require('../../node_modules/three/examples/js/loaders/MTLLoader');  // THREE.MTLLoader
 require('../../node_modules/three/examples/js/loaders/ColladaLoader');  // THREE.ColladaLoader
 require('../../vendor/Raycaster');  // THREE.Raycaster
-require('../../node_modules/three/examples/js/controls/VRControls');  // THREE.VRControls
-require('../../node_modules/three/examples/js/effects/VREffect');  // THREE.VREffect
+require('../../node_modules/webvr-libs/VRControls');  // THREE.VRControls
```
Comment (ngokevin) [inline src/lib/three.js]: we don't need these

https://github.com/aframevr/aframe/issues/1030

File: src/lib/three.js (hunk 1 above)
Comment (cvan) [inline src/lib/three.js]: rebased and tested

File: src/lib/three.js (hunk 3 above)
Comment (ngokevin) [inline src/lib/three.js]: Disregard my last messages :)

**Comment by meta-meta** [comment]
This commit broke mobile VR for me. I'm on Samsung Galaxy Note5. When entering VR mode, the screen goes into landscape, but it's fullscreen, not side by side.

**Comment by masom** [comment]
@meta-meta https://github.com/aframevr/aframe/pull/1065 should fix this.

**Comment by meta-meta** [comment]
Oh cool!

**Comment by dmarcos** [comment]
@meta-meta Thanks for investigating. @cvan Does anything ring a bell?

**Pull request merged**